packages/web-components/src/components/rux-clock/test/clock.spec.ts

```diff
@@ -406,6 +412,34 @@ test.describe('Clock', () => {
         await expect(segment).toContainText('20:47')
     })
 
+    test('can display a static datetime', async ({ page }) => {
+        const template = `<rux-clock static></rux-clock>`
+
+        await page.setContent(template)
+        const el = await page.locator('rux-clock')
+        const segment = await el.locator('.rux-clock__segment__value').nth(1)
+
+        await expect(segment).toContainText('20:02:03')
```

Comment:
testing seconds here is a little squirrely but I cant think of any better way. Ive ran these tests with 30 attempts and didnt see any flake